Makefile

```diff
@@ -75,3 +76,21 @@ push-tag:
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
 	@echo "Pushing tag ${TAG}"
 	@git push git@github.com:open-telemetry/opentelemetry-collector-releases.git ${TAG}
+
+.PHONY: go-licenses
+go-licenses:
+	 go get github.com/google/go-licenses && go build github.com/google/go-licenses && mv go-licenses /usr/bin/
```

Comment:
I'm not sure we want to "contiminate" user linux distros by installing go-licenses into /usr/bin.

Is there anyway to install this somewhere local-ish and reference that local version later?

Reply:
@jsuereth I am not aware that.

Reply:
It should not install under /usr/bin . Otherwise, we'd require sudo to run make, which is not acceptable.

Reply:
@jpkrohling @jsuereth Strongly Agree. Let me re-look into it. How may I make it without putting in /usr/bin

Reply:
You can just install it in the home directory, possibly with `go install github.com/google/go-licenses`. We have something similar for other tools in this repository for ocb, and for other tools as part of the operator repository.

Reply:
@jsuereth @jpkrohling @anuraaga Please review now. I have put go-licenses install in OTELCOL_BUILDER_DIR.